common/init.sls

```diff
@@ -45,8 +45,15 @@ host-{{ hostname }}:
 {% endfor %}
 
 {% for ssh_user in common.ssh_users %}
+{{ ssh_user }}:
+  user.present:
+    - home: /home/{{ ssh_user }}
+    - optional_groups:
+        - wheel
+    - empty_password: True
+
 sshkey-{{ ssh_user }}:
   ssh_auth.present:
```

Comment:
Yup, that should work. Docs: https://docs.saltstack.com/en/2015.5/topics/tutorials/states_pt2.html#call-multiple-states